spec/mailers/subscription_spec.rb

```diff
@@ -45,4 +47,8 @@
     expect(body_lines[12]).to match(/#{I18n.t('subscriptions.email.unsubscribe_text_html')}/)
     expect(body_lines[14]).to match(%r{http:\/\/localhost:3000\/subscriptions\/#{subscription.token}\/unsubscribe})
   end
+
+  def escape_single_quotes(unescaped_string)
```

Comment:
This looks great. Wonder if we can use this method for the other places where we would like to escape the string in our tests.

Reply:
If so we could probably extract this to somewhere that enables us to reuse it without duplicating code

Reply:
Yep.  I'll leave that to be an exercise for a later PR, but agreed we should pull it out if it's needed in multiple places.